### PR TITLE
Shady nagy/net5

### PR DESCRIPTION
**Pull request by ShadyNagy**
Create an appointment added.

### CODE REVIEW

File: FrontDesk/src/FrontDesk.Api/OfficeSettings.cs [hunk 1]
```diff
@@ -6,6 +6,6 @@ namespace FrontDesk.Api
     public class OfficeSettings : IApplicationSettings
     {
         public int ClinicId { get { return 1; } }
-        public DateTime TestDate { get { return new DateTime(2014, 6, 9); } }
+        public DateTime TestDate { get { return new DateTime(2030, 9, 23); } }
```
Comment (ShadyNagy) [inline FrontDesk/src/FrontDesk.Api/OfficeSettings.cs:9]: date changed in api and seed but you need to change the date manually in the appointments page to see the data or we need to create api to get the test date and use there?

Comment (ardalis) [reply FrontDesk/src/FrontDesk.Api/OfficeSettings.cs:9]: Change it manually from what? Does it always default to today? Have it start out on that date, every time.

File: FrontDesk/src/BlazorShared/Models/Appointment/CreateAppointmentRequest.cs [hunk 2]
```diff
@@ -10,9 +10,7 @@ public class CreateAppointmentRequest : BaseRequest
         public int ClientId { get; set; }
         public int RoomId { get; set; }
         public DateTime DateOfAppointment { get; set; }
-        public TimeSpan Duration { get; set; }
```
Comment (ardalis) [inline FrontDesk/src/BlazorShared/Models/Appointment/CreateAppointmentRequest.cs:13]: Don't we need duration?

**Review by ardalis — APPROVED** [review]

File: FrontDesk/src/FrontDesk.Api/OfficeSettings.cs (hunk 1 above)
Comment (ShadyNagy) [reply FrontDesk/src/FrontDesk.Api/OfficeSettings.cs:9]: I changed to force blazor to read from api what is today's date. in PR #5

File: FrontDesk/src/BlazorShared/Models/Appointment/CreateAppointmentRequest.cs (hunk 2 above)
Comment (ShadyNagy) [reply FrontDesk/src/BlazorShared/Models/Appointment/CreateAppointmentRequest.cs:13]: No need as End date calculation in the appointment entity by start and duration from appointment type (UpdateEndTime).

**Pull request merged**